src/defaults.mligo

```diff
@@ -27,15 +21,14 @@ let validate_quorum_threshold_bound (data : initial_config_data) : unit =
 
 let default_config (data : initial_config_data) : config =
   let _ : unit = validate_proposal_flush_expired_level(data) in
-  let _ : unit = validate_max_votes(data) in
```

Comment:
I see you (correctly) removed this check, however can you please still leave a TODO for `#271`, since the problem remains (we need to find safe values for this option)?